packages/typedoc-plugin-appium/lib/model/command-info.ts

```diff
@@ -0,0 +1,18 @@
+import {ExecuteCommandSet, RouteMap} from './types';
+
+/**
+ * Data structure describing routes and commands for a particular module (or project)
+ */
+export class CommandInfo {
+  constructor(
+    public readonly routes: RouteMap,
+    public readonly executeCommands: ExecuteCommandSet = new Set()
```

Comment:
definitely think we should get terminology straight.  don't have a full picture of prior art though.  maybe easier to discuss offline?

Reply:
my vote would be to call them `executeMethods` since that is what we ultimately decided to call them in our docs unless you have a preference/reason otherwise.